main.nf

```diff
@@ -24,27 +25,31 @@ process cBioPortalExport {
          --upload \
          --cbioportal /usr/src/cbioportal \
          --production \
-         --use-grs
+         --use-grs \
+         --cpt_seq_date_replacement_type $cpt_seq_date_replacement_type
```

Comment:
I didn't look super closely is seq_date the only field we replace from main genie?

Reply:
When I looked, it was. Otherwise the main genie clinical files are used to subset out samples and patients from the input data.